ingest/src/sources/mongodb-university/MongoDbUniversityDataApiClient.ts

```diff
@@ -0,0 +1,180 @@
+import fetch from "node-fetch";
+/**
+  Item in the Thought Industries catalog as returned from the
+  MongoDB University Data API.
+ */
+export interface TiCatalogItem {
+  id: string;
+  ti_id: string;
+  /**
+        IDs for videos associated with the catalog item.
+       */
+  associated_videos: string[];
+  /**
+        Learning format for the catalog item.
+        @example "Unit" || "Course" || "Lab" || "...etc"
+       */
+  learning_format: string;
+  name: string;
+  slug: string;
+  status: string;
+  tags: string[];
+  legacy: boolean;
+  /**
+        Thought Industries microsites associated with the catalog item.
+        @example ["University", "Technical Services"]
+       */
+  microsites: string[];
+  /**
+        IDs of labs associated with the catalog item.
+       */
+  associated_labs: string[];
+  // TODO: is this always null?
```

Comment:
To @amanda/alla: can you comment on this question and the other one below on what `in_development` refers to?

Reply:
It means its been tagged by Ops with 'testing', which is something they do while developing a new course

Reply:
and what about the `associated_content` field mentioned in this comment?